datalad/plugin/wtf.py

```diff
@@ -261,6 +262,34 @@ def _describe_location(res):
     }
 
 
+def _describe_credentials():
+    # might later add information on non-keyring credentials
+    props = {}
+    try:
+        import keyring
+        from keyring.util import platform_
+    except ImportError:
```

Comment:
since keyring is part of `install_requires`, I think ImportError for `import keyring` must not happen (silently).  Not sure about `keyring.util.platform_` since not sure if part of "public API".  In either case, I would have logged a warning or at least a DEBUG message so later we could figure out WTF there is no keyring information when expected to be

Reply:
Ah, good point! I removed the try-except handling. `platform_` is a very visible part of the API, I don't think it needs any more special handling.